pluginconfig/pluginconfig-311.yaml

```diff
@@ -183,37 +187,38 @@ versions:
-      router: registry.access.redhat.com/openshift3/ose-haproxy-router:v3.11.146
-      serviceCatalog: registry.access.redhat.com/openshift3/ose-service-catalog:v3.11.146
-      templateServiceBroker: registry.access.redhat.com/openshift3/ose-template-service-broker:v3.11.146
-      webConsole: registry.access.redhat.com/openshift3/ose-web-console:v3.11.146
-      format: registry.access.redhat.com/openshift3/ose-${component}:v3.11.146
+      alertManager: registry.access.redhat.com/openshift3/prometheus-alertmanager:v3.11.153
+      ansibleServiceBroker: registry.access.redhat.com/openshift3/ose-ansible-service-broker:v3.11.153
+      clusterMonitoringOperator: registry.access.redhat.com/openshift3/ose-cluster-monitoring-operator:v3.11.153
+      configReloader: registry.access.redhat.com/openshift3/ose-configmap-reloader:v3.11.153
+      console: registry.access.redhat.com/openshift3/ose-console:v3.11.153
+      controlPlane: registry.access.redhat.com/openshift3/ose-control-plane:v3.11.153
+      grafana: registry.access.redhat.com/openshift3/grafana:v3.11.153
+      kubeRbacProxy: registry.access.redhat.com/openshift3/ose-kube-rbac-proxy:v3.11.153
+      kubeStateMetrics: registry.access.redhat.com/openshift3/ose-kube-state-metrics:v3.11.153
+      node: registry.access.redhat.com/openshift3/ose-node:v3.11.153
+      nodeExporter: registry.access.redhat.com/openshift3/prometheus-node-exporter:v3.11.153
+      oAuthProxy: registry.access.redhat.com/openshift3/oauth-proxy:v3.11.153
+      prometheus: registry.access.redhat.com/openshift3/prometheus:v3.11.153
+      prometheusConfigReloader: registry.access.redhat.com/openshift3/ose-prometheus-config-reloader:v3.11.153
+      prometheusOperator: registry.access.redhat.com/openshift3/ose-prometheus-operator:v3.11.153
+      registry: registry.access.redhat.com/openshift3/ose-docker-registry:v3.11.153
+      registryConsole: registry.access.redhat.com/openshift3/registry-console:v3.11.153
+      router: registry.access.redhat.com/openshift3/ose-haproxy-router:v3.11.153
+      serviceCatalog: registry.access.redhat.com/openshift3/ose-service-catalog:v3.11.153
+      templateServiceBroker: registry.access.redhat.com/openshift3/ose-template-service-broker:v3.11.153
+      webConsole: registry.access.redhat.com/openshift3/ose-web-console:v3.11.153
+      format: registry.access.redhat.com/openshift3/ose-${component}:v3.11.153
       httpd: registry.access.redhat.com/rhscl/httpd-24-rhel7:2.4-104
       masterEtcd: registry.access.redhat.com/rhel7/etcd:3.2.26-16
       genevaLogging: osarpint.azurecr.io/acs/mdsd:master.20190228.1
       genevaStatsd: osarpint.azurecr.io/acs/mdm:git-a909a2e76
       genevaTDAgent: osarpint.azurecr.io/acs/td-agent:master.20190228.1
       azureControllers: quay.io/openshift-on-azure/azure:latest
       canary: quay.io/openshift-on-azure/azure:latest
+      aroAdmissionController: quay.io/makdaam/aro-admission-controller:init-dirty
```

Comment:
(reminder: must fix before merge)

Reply:
Changed to the standard azure image + added AZURE_IMAGE env var override in fakerp.